services/core-web/src/components/common/RenderLargeSelect.js

```diff
@@ -18,14 +18,19 @@ const propTypes = {
     label: PropTypes.label,
     value: PropTypes.string,
   }).isRequired,
-  input: PropTypes.shape({ value: PropTypes.string, onChange: PropTypes.func }).isRequired,
+  input: PropTypes.shape({
+    value: PropTypes.string,
+    onChange: PropTypes.func,
+    onFocus: PropTypes.func,
+  }).isRequired,
   meta: PropTypes.shape({
     touched: PropTypes.bool,
     error: PropTypes.string,
     warning: PropTypes.string,
   }).isRequired,
   handleSearch: PropTypes.func,
   handleSelect: PropTypes.func,
+  handleFocus: PropTypes.func,
 };
 
 const doNothing = () => {};
```

Comment:
![image](https://user-images.githubusercontent.com/17113094/96790059-0a6a3500-13ab-11eb-996e-8a060798f692.png)

Reply:
focus, Luke, focus!

Reply:
It's the `doNothing = () => {};`!12!@!1@!!